support-frontend/assets/pages/contributions-landing/components/PaymentMethodSelector.tsx

```diff
@@ -70,7 +70,6 @@ interface PaymentMethodSelectorProps {
 	isTestUser: boolean;
 	switches: Switches;
 	payPalHasBegunLoading: boolean;
-	amazonPayHasBegunLoading: boolean;
```

Comment:
This has been unused since we removed the amazon pay objects from the redux store (#3919)